resource_obmcs_core_console_history.go

```diff
@@ -110,6 +110,11 @@ func (s *ConsoleHistoryResourceCrud) Get() (e error) {
 	return
 }
 
+func (s *ConsoleHistoryResourceCrud) Delete() (e error) {
+	e = nil
```

Comment:
Makes sense - I agree that there isn't much of a use case here. If it's possible to log a warning here to say that delete isn't supported, that would be good.